spec/factories/movement.rb

```diff
@@ -26,7 +28,7 @@
 
     # This should be far enough in the past so that the offender isn't considered a 'new arrival' by default
     sequence(:movementDate) do |seq|
-      (Time.zone.today - 1.year - seq.days).to_s
+      (Time.zone.today - 5.years + seq.days).to_s
```

Comment:
Additional brackets might make this maths a bit clearer.

Off the top of my head, I have no idea if addition or subtraction is handled first, so seeing a line like this always feels ambiguous to me.

```suggestion
      (Time.zone.today - (5.years + seq.days)).to_s
```